frontend/apps/core-app/src/screens/RecipientListScreen/RecipientListScreen.component.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Button, Text, VStack } from 'native-base';
+import { NavigationProp, RouteProp } from '@react-navigation/native';
+
+import { RootParamList } from '../../navigators/types';
+
+import * as Styles from './RecipientListScreen.styles';
+
+type Props = {
+  route: RouteProp<RootParamList, 'RecipientList'>;
+  navigation: NavigationProp<RootParamList>;
+};
+
+export const RecipientListScreenComponent: React.FC<Props> = ({
+  route,
+  navigation,
+}) => {
+  return (
+    <Styles.Container>
+      <Text variant="display">{route.name}</Text>
+      <VStack space={2} width="sm">
+        <Button
+          variant="major"
+          onPress={() =>
+            navigation.navigate('RecipientProfile', {
+              id: '1',
+            })
+          }
```

Comment:
These should be named functions, but probably alright as is seen as they'll go soon